boards/arm/sensortile_box/Kconfig.defconfig

```diff
@@ -8,11 +8,27 @@ if BOARD_SENSORTILE_BOX
 config BOARD
 	default "sensortile_box"
 
-if SPI
+if BT
```

Comment:
Why not ```depends on BT``` here?

Reply:
`if` is a better option when you have several symbols impacted, as this is the case here.
`depends on` is well suited for single symbol dependency.
So current usage looks good to me.